src/style.css

```
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Medium.ttf);
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 4px;
+  margin-top: 0;
+}
+
+.card__product-code {
+  color: #616070;
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__stars-block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.card__stars_star {
```

```suggestion
.card__stars-star {
```
be consistent in your naming 😉
use either `block__element-name` or `block__element_name`

---

ок, дякую)